code/game/objects/item_materials.dm

```diff
@@ -1,10 +1,10 @@
 /obj/item/on_update_icon()
-	overlays.Cut()
+	cut_overlays()
```

Comment:
How certain are you that you got every /obj/item override of on_update_icon()?

Reply:
Well, most of the things using the old overlay method basically don't call their parent.

But yeah it could mess with stuff's overlays. I could take a deeper look and see if anything in particular does it.